src/components/email-confirm-modal/email-confirm-modal.js

```diff
@@ -0,0 +1,112 @@
+import { Button, Dialog, IconButton } from '@mui/material'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import CloseIcon from '@mui/icons-material/Close'
+import { style } from './email-confirmation-modal.style'
+import { useContext, useEffect, useState } from 'react'
+import imgSuccess from '~/assets/img/email-confirmation-modals/success-icon.svg'
+import imgReject from '~/assets/img/email-confirmation-modals/not-success-icon.svg'
+import { styles } from '~/pages/error/styles/bad-request.styles'
+import LoginDialog from '~/containers/guest-home-page/login-dialog/LoginDialog'
+import { ModalContext } from '~/context/modal-context'
+import { useTranslation } from 'react-i18next'
+import useAxios from '~/hooks/use-axios'
+import { confirmService } from '~/services/confirm-service'
+
+const EmailConfirmModal = ( { confirmToken } ) => {
+  console.log( confirmToken , 'confirmToken inside EmailConfirmModal')
+  const [open, setOpen] = useState(true)
+  const [message, setMessage] = useState()
+  const [statusImage, setStatusImage] = useState(imgSuccess)
+  const [isLoading, setIsLoading] = useState(false)
+  const { setModal } = useContext(ModalContext)
+  const { t } = useTranslation()
+
+  //TODO: useAxios - u need to check what has come from confirmToken
```

Comment:
here i wait for backend part of email confirmation
where i will send query to backand and will get status: error or ok